### ISSUE TITLE
About AArch64 Support.

### ISSUE THREAD

**Issue by longjunyu2** [longjunyu2/MCinaBox#218]
I have updated a project that is related to AArch64 runtime.
If you are interested, [click here](https://github.com/longjunyu2/openjdk-aarch64-jdk8u-androidport)

**Comment by imperialgames13** [comment]
this and for the mcinabox ?? if so how can i put it on my device

**Comment by longjunyu2** [comment]
@imperialgames13 Yes, that project is for mcinabox. In addition, it can also provide available OpenJDK 8 aarch64 to Android devices.

**Comment by misaalanshori** [comment]
How do i use this with MCinabox? Do i need to compile it myself first? And is it required for ARM64 devices?

**Comment by longjunyu2** [comment]
It seems not easy. You need to compile ```OpenJDK 8``` through Android NDK Toolchain on Linux. And then you need compress it , LWJGL and GL4ES with the same file structure with runtime pack. To achieve this, you need also compile ```LWJGL``` and ```GL4ES``` for ```AARCH64```. Then you will get the full runtime for ```AARCH64``` .To make it run on MCinaBox, you also need to edit some codes of MCinaBox, in [```boat/src/main/java/cosine/boat/LoadMe.java```](https://github.com/longjunyu2/MCinaBox/blob/master/boat/src/main/java/cosine/boat/LoadMe.java). More you need also edit the ```gradle``` file to enable building ```native libs``` for ```AARCH64```.

**Comment by imperialgames13** [comment]
sorry I didn't understand anything you said, I don't know if this is easy or difficult, but how can you do it? if yes i will be very grateful

**Comment by misaalanshori** [comment]
Aw man, I don't have much experience with compiling software, especially for Android. Will you be providing the precompiled runtime for ARM64 anytime soon?

**Comment by longjunyu2** [comment]
OK, guys. I will provide the support of aarch64 soon. After I pass the college enterence examination, I will have time to make it work soon. So wait please. :)

**Comment by misaalanshori** [comment]
@longjunyu2 Its okay! Good luck on your exam!

**Comment by imperialgames13** [comment]
ok excellent i am very grateful, good luck in your exam !!

**Comment by Kha-Lik** [comment]
@longjunyu2 hey man, how's your exams? some people still waiting for precompiled binaries)

**Comment by MareGraphics** [comment]
I'll try doing something for ARM architecture
